model/riscv_insts_aext.sail

```diff
@@ -127,7 +127,7 @@ function clause execute(LOADRES(aq, rl, rs1, width, rd)) = {
     /* Get the address, X(rs1) (no offset).
      * Extensions might perform additional checks on address validity.
      */
-    match ext_data_get_addr(rs1, zeros(), Read(Data), width) {
+    match ext_data_get_addr(rs1, zeros(), Read(Data), EXTZ(size_bits(width))) {
```

Comment:
Shouldn't these `size_bits` be `word_width_bytes` everywhere?

Also when you rebase you'll have to upload a load of the vector code that calls `ext_data_get_addr()`. Though you can be lazy and just wait if you want, because I am rebasing/updating this...